build.sh

```diff
@@ -228,6 +235,34 @@ fi
 
 if [[ $_RET != 0 ]]; then
     echo "See error details above. Exit code was $_RET"
+else
+    if [[ $CREATE_DEB != 0 ]]; then
+        DEB_FOLDER=`realpath .`
+        DEB_FOLDER="${DEB_FOLDER}/chakracore_${CREATE_DEB}"
+
+        mkdir -p $DEB_FOLDER/usr/local/bin
+        mkdir -p $DEB_FOLDER/DEBIAN
+        cp $DEB_FOLDER/../ch $DEB_FOLDER/usr/local/bin/
+        if [[ $STATIC_LIBRARY == "" ]]; then
+            cp $DEB_FOLDER/../*.so $DEB_FOLDER/usr/local/bin/
+        fi
+        echo -e "Package: ChakraCore"\
+            "\nVersion: ${CREATE_DEB}"\
+            "\nSection: base"\
+            "\nPriority: optional"\
+            "\nArchitecture: amd64"\
+            "\nDepends: libc6 (>= 2.19)"\
```

Comment:
While we get the ICU story straightened out- don't we need libicu here? Our build instructions also have libuuid/libunwind/liblttng-ust (maybe for the PAL?) should we add them here?